README.md

```diff
@@ -107,7 +107,7 @@ Use the following steps to install the operator and driver, with [GitHub](https:
     1. Download the manifest from GitHub.
 
         ```
-        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.6.0/deploy/installer/generated/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
+        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.7.0/deploy/installer/generated/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
```

Comment:
I think the README should be updated right before GA, so it won't be broken most of the time.
@ArbelNathan what do you think?

Reply:
same, readme update is a separate ticket for GA. right now we want it to be true, especially when `develop` is the default branch and that the readme that everyone sees.

Reply:
Done